ui/react/configuration/reducers.js

```diff
@@ -0,0 +1,246 @@
+import * as actionTypes from './actionTypes';
+import immutable from 'immutable';
+
+const defaultUiState = {
+    nomenclatures: {
+        itemName: [
+            'country',
+            'channel',
+            'region',
+            'city',
+            'organization',
+            'role',
```

Comment:
It doesn't matter for our case. We don't have core or customer modules so we are forced to override rule.item.fetch method on implementation level anyway. This PR is only about adding the possibility to configure what's visible in the front-end